keras_model_specs/models/resnet152.py

```diff
@@ -0,0 +1,185 @@
+# ResNet-152 in Keras was posted by flyyufelix
+# https://gist.github.com/flyyufelix/7e2eafb149f72f4d38dd661882c554a6
+# without a license.
```

Comment:
It's also here: https://github.com/flyyufelix/cnn_finetune/blob/master/resnet_152.py (as MIT)

Reply:
Nice catch ✅